octobot_trading/enums.py

```diff
@@ -364,12 +373,14 @@ class ExchangeConstantsOrderColumns(enum.Enum):
     SYMBOL = "symbol"
     QUANTITY_CURRENCY = "quantity_currency"
     TYPE = "type"
+    OCTOBOT_ORDER_TYPE = "octobot_order_type"
```

Comment:
I don't understand why we should store this value here

Reply:
In my opinion it's not a parser role to do that. To me a parser a is mechanism to convert a format to another, not to implement or prepare internal logic.

Reply:
I removed it